### ISSUE TITLE
DRPM for RPM repo

### ISSUE THREAD

**Issue by jtagcat** [rapid7/metasploit-omnibus#116]
Every day, Metasploit gets a new update, every update is 200-300MB. Across many users that totals to significant bandwidth. This could probably be avoided with DRPM.

## What OS are you running Metasploit on?
Fedora 31
![image](https://user-images.githubusercontent.com/38327267/75243256-731ff400-57c1-11ea-9f3b-a213183b9608.png)

**Comment by adfoster-r7** [comment]
@jtagcat This sounds like a good idea to me; Do you have any insight into what would be required to support this?

**Comment by jtagcat** [comment]
No, I was about to dive in to packaging, but out of time to dive in to yet another topic. Somebody who has packaged the RPM probably has more knowledge about this.

https://lists.fedoraproject.org/archives/list/packaging@lists.fedoraproject.org/ might be of interest, if googling doesn't help.
